src/loadconfig.ts

```diff
@@ -0,0 +1,29 @@
+import store from "./store";
+
+const configs = {};
+
+const uniqueKey = (stringArray: any) => {
+  return stringArray.join("");
+};
+
+export = (composeOptions, defaults, refresh) => {
```

Comment:
please add typedoc comment for this method
also, since this is a new file, please don't do `export =`

do individual export for all new TS code:

```
/**
 * typedoc comments
 * @param ...
 * @returns ...
 */
export const loadConfig() => {
}
```